WinUIGallery/ControlPagesSampleCode/MapControl/MapControlSample_cs.txt

```diff
@@ -0,0 +1,24 @@
+
```

Comment:
Shouldn't this code include setting the servicetoken?

Reply:
Unfortunately we aren't able to provide the service token :(